tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -25,9 +26,18 @@
 # Helper class for storing input cut information
 _InputCutInfo = namedtuple("InputCutInfo", ["name", "shape", "type", "value"], defaults=[None, None, None, None])
 
+
 def is_shape_type(value):
     if isinstance(value, PartialShape):
         return True
+    if 'tensorflow' in sys.modules:
```

Comment:
Do we need to add for PyTroch? It seems to be `torch.Tensor.Size`?

Reply:
`torch.Tensor.Size` represents Python `list`. So it works on master, I added test for this type in [test_complex_params.py](https://github.com/openvinotoolkit/openvino/pull/20009/files#diff-e3878abd42ca567a18e9b90e4bd37a1394a1db1106502b0ee15b4985fc2fb856)